src/test/java/org/broadinstitute/hellbender/utils/test/ArgumentsBuilderTest.java

```diff
@@ -30,9 +31,72 @@ public void testOneBigString(){
 
     @Test
     public void testFromArray(){
-        ArgumentsBuilder args = new ArgumentsBuilder(new Object[]{"Option=" + new File("/path/to"), "OtherOption=" + -1});
-        Assert.assertEquals(args.getArgsArray(), new String[]{"--Option","/path/to", "--OtherOption", "-1"});
+        ArgumentsBuilder args = new ArgumentsBuilder(new Object[]{"Option=" + new File("/path/to"), "OtherOption=" + -1, new File("/somewhere")});
```

Comment:
[FindBugs] INFO: DMI: Hard coded reference to an absolute pathname in org.broadinstitute.hellbender.utils.test.ArgumentsBuilderTest.testFromArray()